ui/src/reducers/tests/draftItem.test.js

```diff
@@ -0,0 +1,218 @@
+import * as actions from "../../actions/draftItem";
+import draftReducer from "../draftItem";
+import { Map, fromJS } from "immutable";
+
+// action draft
+const draft = {
+  access: null,
+  created: null,
+  created_by: null,
+  experiment: null,
+  files: Map({}),
+  id: null,
+  links: null,
+  metadata: {
+    additional_resources: {
+      internal_discussions: "blah blah"
+    },
+    basic_info: {
+      abstract: "This is the abstract",
+      conclusion: "This is the conclusion",
+      ana_notes: ["AN-2121/212"],
+      general_title: "This is the new title"
```

Comment:
`general_title` correct position is under `metadata` ( `metadata > general_title` )